rest/method_get.go

```diff
@@ -58,6 +59,20 @@ func listGet(ctx context.Context, r *http.Request, route *RouteMatch) (status in
 		e = NewError(err)
 		return e.Code, nil, e
 	}
+	// Now we can set Total.
+	list.Total = len(list.Items)
```

Comment:
That is one use case. Your UI may choose not to show the total number of pages or just an estimate (ala Google) or you may use the API for some other purpose. I'm not saying it is not useful, I'm just saying it is not always useful, and I don't think it is wise to have the consumer of the API paying for the computation of this information when not explicitly requested.